views/administration/school.hbs

```diff
@@ -15,16 +15,16 @@
                 <div class="col-sm-12 no-padding">
                     <!-- Admin View -->
                     {{#userHasPermission 'ADMIN_VIEW'}}
-                    <h4>Schule</h4>
+                    <h2 class="h4">Schule</h2>
```

Comment:
Macht es sinn, gleichzeitig ein h2 zu haben und es mit der Klasse h4 zu überschreiben?

Reply:
Ich hatte das so mitbekommen das man die "h"s in Reihenfolge aufbaut und dann mit CSS Klassen überschreibt um Semantisch richtiges HTML zu haben das wiederum Screenreader und  accesibility helfen soll.

Reply:
Da hat @ChrisWeissmann recht. Das HTML sollte semantisch für sich gesehen sinn ergeben und auhc ohne CSS funktionieren. Sämtliches Styling sollte über entsprechende Klassen erfolgen.